src/auto-evo/RunResults.cs

```diff
@@ -18,6 +18,8 @@
 /// </remarks>
 public class RunResults : IEnumerable<KeyValuePair<Species, RunResults.SpeciesResult>>
 {
+    public Dictionary<Patch, Miche> MicheByPatch = new();
```

Comment:
This should be private to avoid other code messing with this in a major way, and to match how this class handles the other results to not have a 50/50 implementation of a POD and a functioning class type.